frontend/packages/client/src/components/Community/CommunityCard/CommunityCard.js

```diff
@@ -0,0 +1,84 @@
+import Blockies from 'react-blockies';
+import { Link } from 'react-router-dom';
+import { useMediaQuery } from 'hooks';
+import JoinCommunityButton from '../JoinCommunityButton';
+import NameAndBody from './NameAndBody';
+import PillsWithStatus from './PillsWithStatus';
+
+/**
+ * CommunityCard will group communities on a row bases,
+ * will use elementsPerRow to determine how many communities to render per row
+ */
+
+const CommunityCard = ({
+  logo,
+  name,
+  body,
+  id,
+  slug,
+  hideJoin,
+  pActive,
+  pPending,
```

Comment:
We need the backend to start returning amount of pending and active proposals on each community